ztp/source-crs/PerformanceProfile.yaml

```diff
@@ -30,3 +30,7 @@ spec:
   # To use the standard (non-realtime) kernel, set enabled to false
   realTimeKernel:
     enabled: true
+  workloadHints:
+      realTime: true
+      highPowerConsumption: false
+      perPodPowerManagement: false
```

Comment:
Looks like the rest of the file is using 2 spaces for the indent - should probably be consistent.